README.md

```diff
@@ -22,9 +22,9 @@ $ ./scripts/package.sh
 This builds the buildpack's source using GOOS=linux by default. You can supply
 another value as the first argument to package.sh.
 
-## `buildpack.yml` Configurations
+## Specifying a project path
 
-There are no extra configurations for this buildpack based on `buildpack.yml`.
-If you would like to specify an `project-path` constraint for the dotnet-build
-buildpack, see its
-[README](https://github.com/paketo-buildpacks/dotnet-core-build/blob/master/README.md#buildpackyml-configurations).
+To specify a project subdirectory to be used as the root of the app, please use
```

Comment:
It'd probably be useful to mention in the documentation that `BP_DOTNET_PROJECT_PATH` specifies the path to your source code app's cs/fs/vbproj file, specifically. Because in some cases, this buildpack runs on FDDs or FDEs where there is no project file _and in those cases, the environment variable has no effect_.